.codespellignore

```diff
@@ -0,0 +1,8 @@
+altho
```

Comment:
I think this is just in a comment that should be "although" unless it's an inside thing I don't get 😆